task2/task2.js

```diff
@@ -0,0 +1,44 @@
+// 2. Transform Streams Homework
+// Task: Create a Node.js script that uses a transform stream to handle JSON objects, modifying each one by
+// adding a new property timestamp and converting it back into a string before writing it to an output file.
+
+const fs = require('fs');
+const { Transform } = require('stream');
+
+// CREATE TRANSFOR STREAM
+const transformedStream = new Transform({
+  readableObjectMode: true,
+  writableObjectMode: true,
+  transform(chunk, coding, callback) {
+    try {
+      const users = JSON.parse(chunk);
```

Comment:
let's try to not use JSON.parse and JSON.stringify, and just work with data as an object. And don't forget to use config `objectMode: true`.